tests/kuttl/tests/adjust-max-concurrency/00-assert.yaml

```diff
@@ -21,19 +21,33 @@ status:
     status: "False"
     type: Ready
   copiedPolicies:
-  - cgu-adjust-max-conc-policy1-common-cluster-version-policy
-  - cgu-adjust-max-conc-policy2-common-pao-sub-policy
+  - cgu-adjust-max-conc-policy1-common-cluster-versio-kuttl
```

Comment:
Would this actually work? The length is already 56, combined with the namespace "default" we'd get "default.cgu-adjust-max-conc-policy1-common-cluster-versio-kuttl" which has a length of 64. Am I missing something here?

Reply:
Length is actually 55